split_data.py

```diff
@@ -10,34 +11,38 @@
 if __name__ == "__main__":
     # Open the raw data file that contains all the rows.
     with open(settings.RAW_DATA_FILE) as f:
-        data = f.read()
+        DATA = f.read()
 
     # Split the data into reviews.
-    reviews = data.split("\n\n")
+    REVIEWS = DATA.split("\n\n")
     # Randomly shuffle the reviews.
-    random.shuffle(reviews)
+    random.shuffle(REVIEWS)
 
     # Downsample the data to run faster
-    reviews = reviews[:40000]
+    REVIEWS = REVIEWS[:40000]
```

Comment:
Why upper case for these?  They aren't constants, and I thought upper case was reserved for those.